docs/src/ToricVarieties/NormalToricVarieties.md

```diff
@@ -86,10 +86,10 @@ affine_open_covering( v::AbstractNormalToricVariety )
 cartier_divisor_group(v::AbstractNormalToricVariety)
 character_lattice(v::AbstractNormalToricVariety)
 class_group(v::AbstractNormalToricVariety)
-map_from_cartier_divisor_group_to_torus_invariant_divisor_group(v::AbstractNormalToricVariety)
-map_from_cartier_divisor_group_to_picard_group(v::AbstractNormalToricVariety)
-map_from_character_to_principal_divisors(v::AbstractNormalToricVariety)
-map_from_weil_divisors_to_class_group(v::AbstractNormalToricVariety)
+map_from_cdivT_to_divT(v::AbstractNormalToricVariety)
+map_from_cdivT_to_pic(v::AbstractNormalToricVariety)
+map_from_char_to_divT(v::AbstractNormalToricVariety)
+map_from_divT_to_cl(v::AbstractNormalToricVariety)
```

Comment:
All these renamings are breaking changes, so if we merge them we have to increase the version to 0.9.0. alternatively use `@deprecate`?


Also, these names are kinda against our conventions. But if they are internal functions only, it should benfine

Reply:
Just so I understand: When you say against the conventions, then you complain about the capital "T"?

I do not understand the point about `breaking changes`. Could you please elaborate?

Reply:
Why do we want to remove these perfectly fine function names with things that are hard to read/parse? I think we are moving away from `idk_wt_divT_or_ cdivT_is` :)

Reply:
Why do we want to remove these perfectly fine function names with things that are hard to read/parse? I think we are moving away from `idk_wt_divT_or_ cdivT_is` :)

Reply:
Why do we want to remove these perfectly fine function names with things that are hard to read/parse? I think we are moving away from `idk_wt_divT_or_ cdivT_is` :)

Reply:
Thank you for elaborating @lkastner.

@thofma The rational for this renaming is to make closer contact with the literature/book. In this particular case, the toric geometry book by Cox-Little-Schenk. In there, the group of torus invariant Weil divisors is denoted as Div_T and that of Cartier torus invariant Weil divisors by CDiv_T. (And similarly Cl for class group and Pic for Picard group.) The hope would be that the new names are easier/more convenient to use.

Indeed, I observed myself that the current function names are somewhat cumbersome. This in turn fits with the observation of @simontelen who brought this issue up in a discussion Monday last week.

Reply:
> Why do we want to remove these perfectly fine function names with things that are hard to read/parse? I think we are moving away from `idk_wt_divT_or_cdivT_is` :)

Oh, the reason in this case was just that these were getting really long and we thought about using the notation from the book. If this is discouraged, we will refrain from doing so.

Reply:
@lkastner you explained it very well, thank you.

But my primary concern is that what @thofma said: in general we are moving towards more self-explanatory names for functions: It is clear to me what `map_from_cartier_divisor_group_to_picard_group` does without looking at the code. Can't say the same for `map_from_cdivT_to_pic`...

You don't really explain the motivation for this; I've just also asked a related question on issue #1036.

The main issue I see with `map_from_cartier_divisor_group_to_picard_group` is that it is really, really long.

Reply:
If I understand it correctly, we try to use the mathematical name (which is hopefully standard in the community) and not the mathematical notation used in a particular reference. As an example, we don't have `cl` instead of `class_group`. I know there some exceptions in Oscar land, but I wish we would not add more.

Maybe this is another case where we would need aliases? Not sure though.

Reply:
I support in general longer names, for example primary_decomposition over primdec. However, if there are well-established shortcuts, known even to beginners in the area, why not use them? For example, everybody is happy with using ext or would someone prefer to replace this by extension_group or some such. The same should apply to pic etc.